blockly-ws-search/README.md

```diff
@@ -0,0 +1,30 @@
+[Home](../README.md)
+
+# blockly-ws-search [![Built on Blockly](https://tinyurl.com/built-on-blockly)](https://github.com/google/blockly)
+
+This package provides an implementation of workspace search for a Blockly workspace.
+
+## Installation
+
+```
+npm install blockly-ws-search
+```
+
+## Usage
+
+```js
+var WorkSpaceSearch = require('blockly-ws-search');
+
+var workspace = Blockly.inject('blocklyDiv' + suffix, options);
```

Comment:
What's a suffix?

Reply:
It comes from multi-workspace demo. It should be removed.
`var workspace = Blockly.inject('blocklyDiv', options);`

Reply:
Also, options is not declared.  Either declare it with a value or add a comment explaining what it is.